src/Statement.php

```diff
@@ -44,49 +45,38 @@ private function __construct(\Doctrine\DBAL\Connection $retriableConnection, Dri
      */
     private function recreateStatement(): void
     {
-        /** @psalm-suppress DeprecatedMethod */
-        $this->stmt = $this->conn->getWrappedConnection()->prepare($this->sql);
+        $ref = new \ReflectionMethod($this->conn, 'connect');
```

Comment:
this is the worstest part. no wrappedConnection and method is protected...

Reply:
Reviewed, it seems the only way. I reverted some changes below in b5b8347ef2be38ee48ecd2e50d252508cf59d565 to minimize the diff because it wasn't useful.